.evergreen/evergreen.yml

```diff
@@ -46,9 +46,9 @@ buildvariants:
     expansions:
```

Comment:
I think you need to update one more spot, the `lib` variant is still using the old versions

Reply:
I just removed them since we don't actually use them!